### PR TITLE
Feature/random cover

### PR DESCRIPTION
**Pull request by jennadilullo8**
# Description

_Created these querySelectors in order to make the functionality of changing the coverImage, title, descriptorOne, and descriptorTwo work;_

var updatedCoverImage = document.querySelector('.cover-image');
var updatedTitle = document.querySelector('.cover-title');
var updatedDescriptorOne = document.querySelector('.tagline-1');
var updatedDescriptorTwo = document.querySelector('.tagline-2');

_Created an eventListener to listen for the window load;_

window.addEventListener('load', generateRandomCover);

_Created the eventHandler when the window loads to generate the random cover, title, and descriptor by accessing the getRandomIndex function and grabbing those elements from their respective arrays. We also updated the cover source and innerTexts of the title and descriptors._

function generateRandomCover() {
  var randomCoverImage = covers[getRandomIndex(covers)];
  var randomTitle = titles[getRandomIndex(titles)];
  var randomDescriptorOne = descriptors[getRandomIndex(descriptors)];
  var randomDescriptorTwo = descriptors[getRandomIndex(descriptors)];
  updatedCoverImage.src = randomCoverImage;
  updatedTitle.innerText = randomTitle;
  updatedDescriptorOne.innerText = randomDescriptorOne;
  updatedDescriptorTwo.innerText = randomDescriptorTwo;
}

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other, Please explain:

## Has This Been Tested?
- [ ] No
- [x] Yes

_We opened the index.html and console to see if the app was working, and it was!!_

# Message for reviewer:

_Completed Iteration 0 with the querySelectors, eventListener, and eventHandler that you see above._

# Checklist:
- [x] My code has no unused/commented out code
- [x] My code has no console logs
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -1,14 +1,17 @@
-// Create variables targetting the relevant DOM elements here 👇
+var updatedCoverImage = document.querySelector('.cover-image');
```
Comment (NicoleLatifi) [inline src/main.js:1]: Great variable creation!

**Comment by NicoleLatifi** [comment]
The variables look good and are using the querySelectors. The eventListener for the window loading is good. And the event handler function will use the getRandomIndex function to pull out a random index for each array and then use that to assign an element to each variable.

File: src/main.js [hunk 2]
```diff
@@ -1,19 +1,25 @@
...
+var updatedTitle = document.querySelector('.cover-title');
+var updatedDescriptorOne = document.querySelector('.tagline-1');
+var updatedDescriptorTwo = document.querySelector('.tagline-2');
```
Comment (rdren0) [inline src/main.js:4]: Good global variable names, make a mental note to come back to these further on in the project and see if they need to be global, or maybe they can be just local.

File: src/main.js [hunk 3]
```diff
@@ -1,19 +1,25 @@
...
 
+var savedCovers = [];
```
Comment (rdren0) [inline src/main.js:6]: I think for now this makes sense to be global. 👍

**Pull request merged**